src/components/pill/pill.spec.js

```diff
@@ -0,0 +1,348 @@
+import React from 'react';
+import 'jest-styled-components';
+import TestRenderer from 'react-test-renderer';
+import { ThemeProvider } from 'styled-components';
+import { shallow, mount } from 'enzyme';
+import Pill from './pill.component';
+import modernConfig from './pill-modern.config';
+import { classicStyleConfig } from './pill-classic.style';
+import { rootTagTest } from '../../utils/helpers/tags/tags-specs/tags-specs';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+import smallTheme from '../../style/themes/small';
+import classicTheme from '../../style/themes/classic';
+import baseTheme from '../../style/themes/base';
+
+const classicStyleTypes = [
+  'default',
+  'disabled',
+  'error',
+  'help',
+  'info',
+  'maintenance',
+  'new',
+  'success',
+  'warning'
+];
+
+const modernStyleTypes = [
+  'neutral',
+  'negative',
+  'warning',
+  'positive'
+];
+
+const modernThemes = [
+  ['small', smallTheme]
+];
+describe('Pill', () => {
```

Comment:
I'd probably add a blank line in between each new `describe` or `it` block so it's more readable